save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt

```diff
@@ -293,9 +291,8 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
             } else {
                 result.getOrThrow()
             }
-            setState {
-                this.project = project
-            }
+            setState { this.project = project }
+
             val headers = Headers().apply {
                 set("Accept", "application/json")
                 set("Content-Type", "application/json")
```

Comment:
We have `jsonHeaders` for that, pls replace it

Reply:
Done